doc/source/exclude_patterns.md

```diff
@@ -0,0 +1,93 @@
+# Exclude Pattern Settings 
+- Optional argument for [in-toto-run.](https://in-toto.readthedocs.io/en/latest/command-line-tools/in-toto-run.html) Path patterns to match paths that should not be recorded as 'materials' or 'products'.  Passed patterns override patterns defined in environment variables or config files (see ```ARTIFACT_EXCLUDE_PATTERNS``` documentation for additional info) 
+- Uses [PathSpec](http://python-path-specification.readthedocs.io) to compile [gitignore-style patterns](https://git-scm.com/docs/gitignore), implementing GitWildMatchPattern class which itself is derived from [Rsync's wildmatch](https://github.com/AndyA/rsync/blob/master/lib/wildmatch.c)   
+- If a directory is excluded, all its files and subdirectories are also excluded
+- Exclude patterns are likely to become command line arguments or part of a config file.
+- Pattern Format
+  - Include backslash (' \\ ') in front of first hash for patterns that begin with a literal hash
+  - Trailing spaces are ignored unless quote with backslash (' \\ ')
+  - Prefix '!' negates the pattern. Excluded patterns will become included again 
+  - The slash (' / ') is used as a directory separator. Separators at beginning or middle (or both) of pattern are relative to directory level of particular .gitignore file. If there is a separator at the end of the pattern then the pattern will only match directories. 
```

Comment:
I wonder if backticks are appropriate here (and elsewhere).

```suggestion
  - The slash (`/`) is used as a directory separator. Separators at beginning or middle (or both) of pattern are relative to directory level of particular .gitignore file. If there is a separator at the end of the pattern then the pattern will only match directories. 
```

Reply:
Do you recommend to include the said characters in quotes, without quotes, or just remove the said characters all together?